package io.github.cni274.orderservice.exception;

public class UserException extends RuntimeException {
}
